esp-metadata/src/lib.rs

```diff
@@ -407,9 +407,7 @@ impl Config {
                     .properties()
                     .filter_map(|(name, value)| match value {
                         Value::Boolean(true) => Some(vec![name.to_string()]),
-                        Value::NumberList(values) => {
-                            Some(values.iter().map(|val| format!("{name}_{val}")).collect())
-                        }
+                        Value::NumberList(_) => None,
```

Comment:
RSA quickly proved that this wasn't the best idea, lol, we don't want 200 symbols just for the multiplication sizes.